frontend/src/pages/ActivityReport/index.css

```diff
@@ -1,13 +1,29 @@
-.smart-hub--form-section {
-  margin-top: 2rem;
+.smart-hub--form-group {
+  margin-top: 1.5rem;
 }
 
 .smart-hub--report-checkbox {
   margin-top: 1.25rem;
 }
 
+.smart-hub--form-section {
+  margin-top: 2rem;
```

Comment:
[`.margin-top-4`](https://designsystem.digital.gov/utilities/margin-and-padding/)